### PR TITLE
Indonesian Taiko Showdown Wiki Page

### PR DESCRIPTION
**Pull request by fajar13k**
- Add IDTS to Tournaments index page
- Add IDTS Wiki page

---

### CODE REVIEW

File: wiki/Tournaments/ATS/1/en.md [hunk 1]
```diff
@@ -0,0 +1,365 @@
+---
+tags:
+- Indonesian Taiko Showdown
+- IDTS
+---
+# Indonesian Taiko Showdown
+
+![IDTS Logo](img/logo.png)
+
+The **Indonesian Taiko Showdown** ***(IDTS)*** was an Indonesian double elimination one-on-one osu!taiko tournament hosted by ![][flag_ID] [fajar13k](/users/7100002). It was the 1st installment of the tournament.
```
Comment (Joehuu) [inline wiki/Tournaments/ATS/1/en.md]: `***(IDTS)***` -> `(***IDTS***)`

File: wiki/Tournaments/ATS/1/en.md [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+| Designers | ![][flag_PH] [OsuMe65](/users/852867), ![][flag_ID] [fajar13k](/users/7100002) |
+| Referees | ![][flag_ID] [fajar13k](/users/7100002), ![][flag_ID] [LittleBird-](/users/6668671), ![][flag_ID] [MRaihanZ](/users/6668671), ![][flag_ID] [Etherealize](/users/7850608), ![][flag_ID] [Niko-nyan](/users/906991), ![][flag_ID] [Aozora-](/users/6918206) |
+| Statistician | ![][flag_ID] [fajar13k](/users/7100002) |
+| Wiki | ![][flag_ID] [fajar13k](/users/7100002) |
+
+## Links
+
+- [Discussion thread](/community/forums/topics/576070)
+- [Livestream](https://www.twitch.tv/osutaikoshowdown)
+- [Indonesian Taiko Showdown Skins](https://puu.sh/whPiI/0fd221d5fa.zip)
```
Comment (Joehuu) [inline wiki/Tournaments/ATS/1/en.md]: Lowercase `Skins`

File: wiki/Tournaments/ATS/1/en.md [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![][flag_ID] [pmriva](/users/2180885) |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | ![][flag_ID] [Nishizumi](/users/2496768) |
+
+-----------------------
+
+## Mappools
+
+### Finals
+
+**This mappool was used during Finals and Grand Finals**
```
Comment (Joehuu) [inline wiki/Tournaments/ATS/1/en.md]: End sentence with a period.

File: wiki/Tournaments/en.md [hunk 4]
```diff
@@ -94,7 +94,7 @@ Unofficial tournaments/competitions hosted by the communities.
 
 | Name | Start | End | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") |
 | --- | --- | --- | --- | --- | --- |
-| Indonesian Taiko Showdown 2017 | 2017-05-06 | 2017-06-11 | ![][flag_ID] [XK2238](/users/1139209) | ![][flag_ID] [pmriva](/users/2180885) | ![][flag_ID] [Nishizumi](/users/2496768) |
+| [Indonesian Taiko Showdown](/wiki/Tournaments/ATS/1/) | 2017-05-06 | 2017-06-11 | ![][flag_ID] [XK2238](/users/1139209) | ![][flag_ID] [pmriva](/users/2180885) | ![][flag_ID] [Nishizumi](/users/2496768) |
```
Comment (Joehuu) [inline wiki/Tournaments/en.md]: No slash(`/`) in the end of links. I see five instances, but idk if it should be fixed..

Comment (fajar13k) [reply wiki/Tournaments/en.md]: I will delete the slash in each end of links.

File: wiki/Tournaments/ATS/1/en.md [hunk 5]
```diff
@@ -0,0 +1,365 @@
...
+
+### Quarterfinals
+
+| Saturday, 2017-05-20 |  |  |  |  | 
+| --: | :-: | :-- | :-- | :-: |
+| **Katdon\_donKat** ![][flag_ID] | **5** | 1 | ![][flag_ID] \_Destroyer\_ | [#1](/community/matches/33286726) |
+| **steven1** ![][flag_ID] | **5** | 0 | ![][flag_ID] Buyut | -win by default- |
+| **Nishizumi** ![][flag_ID] | **5** | 0 | ![][flag_ID] Lightning Wyvern | [#1](/community/matches/33289527) |
+
+
```
Comment (Joehuu) [inline wiki/Tournaments/ATS/1/en.md]:
```suggestion

```

**Pull request merged**